roles/apim-gateway/templates/carbon-home/repository/conf/tomcat/catalina-server.xml.j2

```diff
@@ -75,8 +75,8 @@
                    compressionMinSize="2048"
                    noCompressionUserAgents="gozilla, traviata"
                    compressableMimeType="text/html,text/javascript,application/x-javascript,application/javascript,application/xml,text/css,application/xslt+xml,text/xsl,image/gif,image/jpg,image/jpeg"
-                   keystoreFile="${carbon.home}/repository/resources/security/wso2carbon.jks"
-                   keystorePass="wso2carbon"
+                   keystoreFile="${carbon.home}/{{ key_store_location }}"
```

Comment:
The whole path should be parameterized. If the user adds the key file to an outside location, `${carbon.home}` path prefix will give you an error.